mungegithub/mungers/sig-mention-handler.go

```diff
@@ -107,7 +107,7 @@ func (s *SigMentionHandler) Munge(obj *github.MungeObject) {
 There are no sig labels on this issue. Please [add a sig label](https://github.com/kubernetes/test-infra/blob/master/commands.md) by:
 	
 1. mentioning a sig: `+"`@kubernetes/sig-<group-name>-<group-suffix>`"+`
-    e.g., `+"`@kubernetes/sig-api-machinery-<group-suffix>`"+` to notify the API Machinery sig, OR
+    e.g., `+"`@kubernetes/contributor-experience-<group-suffix>`"+` to notify the contributor experience sig, OR
```

Comment:
I think the `sig-` part was left out from `@kubernetes/contributor-experience-<group-suffix>`